crates/proto/src/xfer/dns_multiplexer.rs

```diff
@@ -72,13 +71,14 @@ impl ActiveRequest {
     }
 
     /// Returns true of the other side canceled the request
+    // FIXME: change to is_closed?
```

Comment:
For an `ActiveRequest`, I feel like `is_cancelled()` makes more semantic sense than `is_closed()`.